lib/validators/movieList.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  sort: Joi.string().valid(['id', 'title', 'release_year', '-id', '-title', '-release_year']).optional().default('id'),
+  q: Joi.string().optional().default('').lowercase()
```

Comment:
@elnaz if the q makes it through as 'undefined', the result set was always 0 for me 😞